gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/upgrade/upgrader/DefaultEnvironmentUpgrader.java

```diff
@@ -28,18 +29,18 @@
  * @author GraviteeSource Team
  */
 @Component
-public class DefaultEnvironmentInitializer implements Initializer {
+public class DefaultEnvironmentUpgrader implements Upgrader {
```

Comment:
I think we should change the order to ensure this upgrader runs after the `DefaultOrganization`

Reply:
And also adapt the order of existing upgraders

Reply:
forget my comment, we will check the order of all upgraders/initializers in a dedicated PR